flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/catalog/QueryOperationCatalogView.java

```diff
@@ -28,7 +28,7 @@
  * A view created from a {@link QueryOperation} via operations on {@link org.apache.flink.table.api.Table}.
  */
 @Internal
-public class QueryOperationCatalogView extends AbstractCatalogView {
+public class QueryOperationCatalogView extends CatalogViewImpl {
```

Comment:
Previously QueryOperationCatalogView, HiveCatalogView, and GenericCatalogView were children of AbstractCatalogView. Now we combine the latter two, and make the first one as a child of the combo, which might be a concern. I'm not sure if this posts any issue, but maybe we can keep the relationship unchanged?

Reply:
Good point! I missed this since its counterpart of table is named `ConnectorCatalogTable`, maybe this class should be named `ConnectorCatalogView` :) .... nvm

I will restore AbstractCatalogView

Reply:
I think we need to revert changes to this file.

Reply:
right, how could I forget that...